Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM golang:1.19.3-alpine AS builder
 
 WORKDIR /app
 COPY . /app
-RUN CGO_ENABLED=0 go build -trimpath -ldflags "-s -w" -v -o atlantis .
+RUN --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
```

Comment:
Have you been able to test this in your fork to confirm if this works as intended?

Reply:
I have tested it locally and use the same pattern in a bunch of other repositories which also build using GitHub Actions.

Reply:
For this to have any effect in CI here, though, I believe it needs the caching options from my other PR, otherwise the cache won't be shared between the runs.